clang/test/CheckedCRewriter/itypecast.c

```diff
@@ -0,0 +1,45 @@
+// Tests for Checked C rewriter tool.
+//
+// Checks cast insertion while passing arguments to itype parameters.
+//
+// RUN: cconv-standalone %s -- | FileCheck -match-full-lines %s
+
+int foo(int **p:itype(_Ptr<_Ptr<int>>));
+int bar(int **p:itype(_Ptr<int *>));
+int baz(int ((*compar)(const int *, const int *)) :
+             itype(_Ptr<int (_Ptr<const int>, _Ptr<const int>)>));
+             
+int func(void) {
+    int ((*fptr1)(const int *, const int *));
+    int ((*fptr2)(const int *, const int *));
```

Comment:
It's odd to make local variables for these  and then call them without assigning anything to them? Maybe make them parameters to `func` ?